Editor/CKEditor/Resources/public/js/CKEditor.url.js

```diff
@@ -1,2 +1,2 @@
 // Make sure to work with jcss combination enabled.
-var CKEDITOR_BASEPATH = Zikula.Config.baseURI + '/web/editors/ckeditor/ckeditor/';
+var CKEDITOR_BASEPATH = Zikula.Config.baseURI + '/modules/Scribite/Editor/CKEditor/vendor/ckeditor/';
```

Comment:
Assets should be loaded from `web`. Scribite 6 cares for copying them automatically.

Reply:
Interesting, hope this procedure cares about possible updates.

Reply:
Yes it checks if the latest version exists in `web`.

See https://github.com/zikula-modules/Scribite/blob/master/Editor/Factory.php#L99
and https://github.com/zikula-modules/Scribite/blob/master/Helper/AssetHelper.php#L41

Reply:
For me, any duplication of code is the cause of possible mistakes and misunderstandings.
This can be avoided if the editors are just in the right place.
An additional problem is too many files on the server, especially in shared hosting.

Reply:
Background is that only `web` should be accessible from the public, while core and extensions can live elsewhere (e.g. at a central place on the server).

Reply:
This is understandable.
The structure of the module may be just:
web/editors/.....
modules/Scribite/....

Reply:
The structure of the module may be for example:
`/usr/share/zikula/modules/Zikula/ScribiteModule/...`
While sites use something like:
`/var/www/html/site1/web/editors/...`
`/var/www/html/site2/web/editors/...`
`/var/www/html/site3/web/editors/...`